sentry-android-okhttp/src/main/java/io/sentry/android/okhttp/SentryOkHttpInterceptor.kt

```diff
@@ -57,6 +57,7 @@ class SentryOkHttpInterceptor(
             .addPackage("maven:io.sentry:sentry-android-okhttp", BuildConfig.VERSION_NAME)
     }
 
+    @Suppress("LongMethod")
```

Comment:
Did it start erroring?

Reply:
yeah on feat/7.0.0 branch, but I fixed it here by accident. Doesn't matter I guess, after we merge it